tests/purs/layout/BacktickOperator.purs

```diff
@@ -0,0 +1,16 @@
+module Test where
+
+example1 = do
+  foo bar
+  <|> baz
+
+example2 = do
+  foo bar
+  `wat` baz
+
+example3 =
+  case _ of
+    Foo a -> 1
+    Bar b -> 2
+  `append` 3
```

Comment:
I think this should be indented to the same level as the case branches, not the case keyword.

Reply:
Ah... Done.